pose_format/utils/openpose.py

```diff
@@ -157,25 +158,46 @@ def limbs_index(limbs: List[Tuple[str, str]], points: List[str]) -> List[Tuple[i
 ]
 
 
-def load_openpose(frames: list, fps: float = 24, width=1000, height=1000, depth=0):
+OpenPoseFrame = Dict[str, Any]
+OpenPoseFrames = Dict[int, OpenPoseFrame]
```

Comment:
I believe that in the case of `dgs` the keys are actually strings `"4321"`

Reply:
Yes they are strings, but I was planning to modify the DGS dataset loader to convert them to ints. Is there a reason you would rather not do that?

Reply:
No reason, that would also be OK if you change it on the DGS side!